Makefile

```diff
@@ -0,0 +1,77 @@
+.PHONY: help
+help:
+	@echo "---------------HELP-----------------"
+	
+	@echo "- make build --> Create docker image with dependencies needed for development"
+	@echo "- make run --> Execute timetracker_ui docker container"
+	@echo "- make stop --> Stop container timetracker_ui"
+	@echo "- make restart --> Restart container timetracker_ui"
+	@echo "- make remove --> Delete container timetracker_ui"
+	@echo "- make test --> Run all tests on docker container timetracker_ui"
+	@echo "- make publish --> Publish the container image timetracker_ui"
+
+	@echo "- make build_prod --> Create docker image with dependencies needed for production"
+	@echo "- make run_prod --> Execute timetracker_ui_prod docker container"
+	@echo "- make remove_prod --> Delete container timetracker_ui_prod"
+	@echo "- make publish_prod --> Publish the container image timetracker_ui_prod"
+
+	@echo "- make login --> Login in respository of docker images"
+	@echo "------------------------------------"
+
+.PHONY: build
+build:
+	-docker rmi timetracker_ui
+	docker-compose build
+
+.PHONY: run
+run:
+	docker-compose --env-file ./.env up -d
+	docker logs -f timetracker_ui
```

Comment:
Extract `docker logs` to another make task named as `logs`

Reply:
Done